src/flows/recommendation_api/corpus_candidate_sets/collections_by_recency.py

```diff
@@ -0,0 +1,49 @@
+from prefect import Flow
+
+from api_clients.pocket_snowflake_query import PocketSnowflakeQuery, OutputType
+from common_tasks.corpus_candidate_set import (
+    create_corpus_candidate_set_record,
+    load_feature_record,
+    feature_group,
+    validate_corpus_items,
+)
+from utils import config
+from utils.flow import get_flow_name, get_interval_schedule
+
+FLOW_NAME = get_flow_name(__file__)
+
+RECENT_COLLECTIONS_CANDIDATE_SET_ID = "92af3dae-25c9-46c3-bf05-18082aacc7e1"
+
+EXPORT_COLLECTIONS_CANDIDATE_SET_SQL = """
+SELECT 
+    approved_corpus_item_external_id as "ID", 
+    topic as "TOPIC"
+FROM "ANALYTICS"."DBT"."APPROVED_CORPUS_ITEMS"
```

Comment:
if you want the collections most recently scheduled for the new tab you would probably need the `SCHEDULED_CORPUS_ITEMS` model.  not sure how much the results would differ